src/Process/Deploy/InstallUpdate/ConfigUpdate/SearchEngine/ElasticSearch.php

```diff
@@ -69,13 +87,11 @@ public function getVersion(): string
             $esConfig = $relationships['elasticsearch'][0];
```

Comment:
use new methods you added getConfig() and check all other places where need to change

Reply:
It is out of scope of my task, I'll create tech debt